packages/azure-kusto-data/tsconfig.json

```diff
@@ -1,13 +1,25 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src/**/*.ts", "test/**/*.ts","webpack.config.ts"],
+  "include": [
+    "src/**/*.ts",
+    "test/**/*.ts",
+    "src/**/*.json",
+    "test/**/*.json",
+    "package.json"
```

Comment:
why package.json

Reply:
We import package.json for the version, so ts need to know about it.